extensions/default/src/ViewerLayout/ResizablePanelsHook.tsx

```diff
@@ -43,15 +27,15 @@ const useResizablePanels = (
   setRightPanelClosed
 ) => {
   const [leftPanelExpandedWidth, setLeftPanelExpandedWidth] = useState(
-    sidePanelExpandedDefaultWidth
+    panelGroupDefinition.left.initialExpandedWidth
   );
   const [rightPanelExpandedWidth, setRightPanelExpandedWidth] = useState(
-    sidePanelExpandedDefaultWidth
+    panelGroupDefinition.right.initialExpandedWidth
   );
-
-  // Percentage sizes.
-  const [resizablePanelCollapsedSize, setResizablePanelCollapsedSize] = useState(0);
-  const [resizablePanelDefaultSize, setResizablePanelDefaultSize] = useState(0);
+  const [leftResizablePanelMinimumSize, setLeftResizablePanelMinimumSize] = useState(0);
+  const [rightResizablePanelMinimumSize, setRightResizablePanelMinimumSize] = useState(0);
+  const [leftRizeablePanelCollapsedSize, setLeftResizePanelCollapsedSize] = useState(0);
```

Comment:
*typo this should be leftResizablePanelCollapsedSize. I believe there is an extension for VS code that will help for spell checking. It is called Code Spell Checker. Comes in handy.

Reply:
thanks spelling police